packages/api/src/index.tsx

```diff
@@ -0,0 +1,5 @@
+// Please include any changes you make to this file to tsconfig.json files array
```

Comment:
We don't need this comment.

`tsconfig.json/files` lists all entrypoints and TSC will crawl thru the tree from these entrypoints.